client/scss/layouts/_login.scss

```diff
@@ -65,6 +65,11 @@
       max-width: calc(theme('spacing.32') * 3.5);
       padding: theme('spacing.12') theme('spacing.14');
     }
+
+    // border for windows high contrast mode, light
+    @media (forced-colors: active) {
+      border: 5px solid transparent;
```

Comment:
Just a thought. Should we use `outline` instead. This is not a focusable element so the outline won't be used here. Outline also helps for avoiding additional content size changes. 

https://developer.mozilla.org/en-US/docs/Web/CSS/outline